### ISSUE TITLE
Move parts of Islandora Demo Feature into Core Feature

### ISSUE THREAD

**Issue by seth-shaw-unlv** [Islandora-CLAW/CLAW#866]
As I develop my own content models, the media parts of the Demo feature should be generally available via the Core so I don't have to recreate them all in my own module:

- Actions:
  - Generate a service file from image preservation master
  - Generate a thumbnail from an image service file
- Context:
  - Binary
  - Files
  - Image Preservation Master
  - Image Service File
  - Media
  - Taxonomy Terms
- Entity view display:
  - media.file.open_seadragon
  - media.image.open_seadragon
- Field
  - Member of
  - Tags (although I would be in favor of giving this a more islandora-specific name)

**Comment by dannylamb** [comment]
When splitting things out I may have been too aggressive trying to keep core lean. And as a consequence, the rest getting lumped together as 'demo' probably isn't granular enough considering how optional everything really is.  Houdini, Cantaloupe, and lots of other stuff may not be deployed even though we're assuming it exists in our vagrant installs.  And folks might have just pngs or just tiffs, and wouldn't need to use `context` to switch display modes.

So yeah... it's all jumbled up.  And trying to slice it all apart may be difficult with config dependencies.  But the Media fields definitely need to go into core.  And it would be a good opportunity to evaluate the name for the tags field.  I'm also thinking the demo image actions should probably ship with `islandora_image` as a feature, but again, dependencies might prove problematic.

**Comment by ajs6f** [comment]
@dannylamb, as an example that I think (but am not sure) is on point: at our site we now have completely separate systems and teams of people handling things like IIIF image delivery and the kind of Drupal frontend CLAW has. I can't imagine us deploying any of CLAW's components that would re-create an extant functional service.

**Comment by dannylamb** [comment]
TL;DR Start by moving the media "Media Of" and "Tags" fields over to the core feature, and we'll sort out the rest as we get to it.
